Automattic/jetpack
# AI Excerpt: re-enable AI Excerpt panel

This pull request aims to reactivate the AI Excerpt panel's functionality within the front-end context of P2 sites. To achieve this goal, the following steps have been taken:

- The Content Lens extension has been relocated to the `.extensions/plugins` folder. It will continue to extend the editor app with the AI excerpt feature in the block editor context. However, it will be removed silently when it's in the P2 front-end context. This change won't affect the AI assistant block, as they are now independent codes.
- The AI Excerpt feature's availability still depends on the AI Assistant block. If the user disables the block, the feature will also be disabled.


Fixes #

## Proposed changes:

* AI Excerpt: Re-enable the AI Excerpt panel

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Confirm the AI Excerpt is there as long as the `beta` extensions are enabled

<img width="558" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/1e5b26e1-7f8f-4cb6-97e1-109894da76f3">

* Apply these changes in a P2 testing site
* Confirm the AI Assistant block is available and works as expected.

<img width="985" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/70626e31-ddd1-41e7-8c21-bb6683e55e36">

* Disable the AI Excert block from the app preferences:
<img width="778" alt="Screenshot 2023-09-12 at 09 53 10" src="https://github.com/Automattic/jetpack/assets/77539/0f4c0244-b3ec-4d54-b7bd-dd202ecf82f5">

* Confirm the AI Excerpt extension is not available.
* Confirm you see the core Excerpt panel

<img width="457" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/dfc84875-ec66-4b47-829c-1766c84436a2">
